### PR TITLE
Replace More Uses of Super Globals

### PR DESCRIPTION
**Pull request by tn3rb**
This PR fixes some uses of the PHP Superglobals that were missed in #3478

### CODE REVIEW

File: core/admin/EE_Admin_List_Table.core.php [hunk 1]
```diff
@@ -863,4 +863,12 @@ protected function _action_string(
         $content .= ! empty($action_container) ? '</' . $action_container . '>' : '';
         return $content;
     }
+
+
+    protected function getReturnUrl()
+    {
+        $host       = $this->_admin_page->get_request()->getServerParam('HTTP_HOST');
+        $uri        = $this->_admin_page->get_request()->getServerParam('REQUEST_URI');
```
Comment (knazart) [inline core/admin/EE_Admin_List_Table.core.php:871]:
```suggestion
        $host = $this->_admin_page->get_request()->getServerParam('HTTP_HOST');
        $uri  = $this->_admin_page->get_request()->getServerParam('REQUEST_URI');
```

File: modules/gateways/Invoice/lib/invoice_functions.php [hunk 2]
```diff
@@ -1,59 +1,63 @@
...
+            // then store it for later display
+            if ($fname !== '.'
+                && $fname !== 'index.php'
+                && $fname !== '..'
+                && $fname !== '.svn'
+                && $fname !== basename($request->getServerParam('PHP_SELF'))
+                && $fname !== '.DS_Store'
+                && $fname !== 'images'
+                && $fname !== 'print') {
+                // store the filename
```
Comment (knazart) [inline modules/gateways/Invoice/lib/invoice_functions.php:36]:
```suggestion
                && $fname !== 'print'
            ) {
                // store the filename
```

**Review by knazart — APPROVED** [review]
👍

File: core/admin/EE_Admin_List_Table.core.php (hunk 1 above)
Comment (tn3rb) [reply core/admin/EE_Admin_List_Table.core.php:871]: thnx, will fix

File: modules/gateways/Invoice/lib/invoice_functions.php (hunk 2 above)
Comment (tn3rb) [reply modules/gateways/Invoice/lib/invoice_functions.php:36]: thnx, will fix

**Pull request merged**